core/src/Nav/Node/Node.less

```diff
@@ -78,10 +87,14 @@
 }
 
 .selected {
-  background-color: #c3cddf;
+  background-color: darken(@zesty-light-blue, 6%);
   opacity: 1;
   a {
-    color: #000;
+    color: @white;
+    font-weight: 600;
+    svg {
+      opacity: 1;
```

Comment:
Why are we setting an opacity of 1? That doesn't allow any blend of the background?